bin/create

```diff
@@ -23,14 +23,15 @@ var create = require('./lib/create');
 var args  = require('./lib/simpleargs').getArgs(process.argv);
 
 if (args['--help'] || args._.length === 0) {
-    console.log('Usage: ' + path.relative(process.cwd(), path.join(__dirname, 'create')) + ' <path_to_new_project> <package_name> <project_name> [<template_path>] [--link]');
+    console.log('Usage: ' + path.relative(process.cwd(), path.join(__dirname, 'create')) + ' <path_to_new_project> <package_name> <project_name> <activity_name> [<template_path>] [--link]');
```

Comment:
Shifting the `<template_path>` arg will break scripts that currently use it. Maybe use `--activity-name=` instead?